Sources/OpenLogin/WebAuth.swift

```diff
@@ -47,75 +91,115 @@ public class WebAuth: NSObject {
 
      - parameter callback: Callback called with the result of the WebAuth flow.
      */
-    public func start(_ callback: @escaping (Result<State>) -> Void) {
-        guard
-            let bundleId = Bundle.main.bundleIdentifier,
-            let redirectURL = URL(string: "\(bundleId)://openlogin")
-        else { return callback(.failure(WebAuthError.noBundleIdentifierFound)) }
+    public func login(_ loginParams: OLLoginParams, _ callback: @escaping (Result<OpenLoginState>) -> Void) {
+        DispatchQueue.main.async { [self] in
+            guard
+                let bundleId = Bundle.main.bundleIdentifier,
+                let redirectURL = URL(string: "\(bundleId)://openlogin")
```

Comment:
pls use the same url structure for this as android.
That way in react native, they can define only one redirect possibly